tests/namespace_test/NamespaceA/SecondTableInA.py

```diff
@@ -4,6 +4,7 @@
 
 import flatbuffers
 from flatbuffers.compat import import_numpy
+from tests.namespace_test.NamespaceC.TableInC import TableInC, TableInCT
```

Comment:
this is generated code.. these changes will need to be made in the code generator to have them stick.